### DISCUSSION TITLE
prosit setting and decoy output

### DISCUSSION THREAD

**Discussion by ghost** [vdemichev/DiaNN#229] [Q&A] [answered]
Hi Vadim,

Thanks for your effort in developing the impressive and excellent software DIA-NN, It helps a lot with my work.

And I have two questions.

1.
It seems that DIA-NN used built-in Prosit to predict spectrums and I notice all collision_energy in (Prosit input from FASTA) of peptides were set as 30, but online Prosit version was set with different collision_energy.  
https://www.proteomicsdb.org/prosit/

is that ok? Now the performance of the library-free mode is good enough. Will it even have better performance if we set the "correct energy" in prosit input?   it's very intresting.

And may I ask could you share with me the built-in prosit software?  Because I found it's hard to set the environment to run prosit:(.   maybe  I need to pay more time and patience. 

2.
I found there is no decoy peptide or precursor list in the result.tsv.  Just a suggestion, with decoy terms listed in tsv,  I think users could calculate the protein FDR or precursor FDR by themself and there will be more flexibility in the data processing.

Thanks again for your contribution to proteomics!

best,
Yuanliang

**Comment by vdemichev** [comment, marked as answer]
Hi Yuanliang,

No, DIA-NN does not use Prosit, DIA-NN's own predictor is very different from Prosit.
There's an option to generate Prosit input in DIA-NN, solely as a convenience for those who might want to use Prosit for whatever reason, but we recommend using DIA-NN's own predictor instead.

Yes, you can use DIA-NN as a standalone predictor. It generates a .predicted.speclib file out of a FASTA database or a spectral library. The .predicted.speclib file is a spectral library and can be converted, using DIA-NN, to a simple text (.tsv) format.

DIA-NN uses a very good algorithm for FDR calculation, adjusting it would probably be deleterious to the performance.

Best,
Vadim

**Comment by ghost** [comment]
Hi Vadim,

Thanks for your prompt reply.

And I noticed that this own predictor has not been introduced and demonstrated on the nature method paper. I am a little curious why because it's a really important part of DIA-NN for library-free search in my view.

Thank you for taking the time to respond to my queries; it is really appreciated!

Best,
Yuanliang

**Reply by vdemichev** [reply]
The predictor was implemented in the later versions, not the one in the Nature Methods paper. In general, it's just a predictor which is well implemented technically, but that has little conceptual novelty and thus there's hardly any point in publishing it in its current state.

Best,
Vadim
